### PR TITLE
support laravel 11

### PR DESCRIPTION
**Pull request by atmonshi**
😉

### CODE REVIEW

**Comment by awcodes** [comment]
Wanna update the tests for L11 too, please?

**Comment by atmonshi** [comment]
opps I always forget about the git workflow

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -21,13 +21,25 @@ jobs:
...
         os: [ubuntu-latest, windows-latest]
-        php: [8.2, 8.1]
-        laravel: [10.*]
+        php: [8.3, 8.2, 8.1]
+        laravel: [11.*, 10.*]
         stability: [prefer-lowest, prefer-stable]
         include:
           - laravel: 10.*
             testbench: 8.*
             carbon: ^2.63
```
Comment (awcodes) [inline .github/workflows/tests.yml:30]: Should these be '2.*' and '3.*' instead?

**Comment by atmonshi** [comment]
still the test falling not sure why
https://github.com/atmonshi/scribble/actions/runs/8393025706/job/22987062051?pr=2

I feel it some incompatible versions not sure which one!

**Comment by awcodes** [comment]
Thanks

**Pull request merged**